core/tests/webdriverio_utils/workflow.js

```diff
@@ -125,13 +125,10 @@ var publishExploration = async function() {
 
   let width = (await browser.getWindowSize()).width;
   if (width > 768) {
-    await waitFor.elementToBeClickable(publishButton);
-    await publishButton.isDisplayed();
```

Comment:
They are not needed because we are using `action.click` which by defaults wait for the button to be clickable before clicking on it.